lib/gui/app/pages/main/Flash.tsx

```diff
@@ -132,7 +116,7 @@ async function flashImageToDrive(
 }
 
 const formatSeconds = (totalSeconds: number) => {
-	if (!totalSeconds && !_.isNumber(totalSeconds)) {
+	if (!totalSeconds && Number.isNaN(totalSeconds)) {
```

Comment:
`!_isNumber` is not equivalent to `isNaN`.

`formatSeconds` only accepts number according to the typings, if this is not true you want something like:

```typescript
if (typeof totalSeconds !== 'number' || !isFinite(totalSeconds)) {
    return '';
}
```
```